zenoh/src/lib.rs

```diff
@@ -199,24 +200,42 @@ pub mod session {
     #[doc(hidden)]
     pub use crate::api::session::init;
     pub use crate::api::session::open;
+    #[zenoh_macros::unstable]
+    #[doc(hidden)]
+    pub use crate::api::session::InitBuilder;
+    pub use crate::api::session::OpenBuilder;
     pub use crate::api::session::Session;
     pub use crate::api::session::SessionDeclarations;
     pub use crate::api::session::SessionRef;
+    pub use crate::api::session::Undeclarable;
+}
+
+/// Tools to access information about the current zenoh [`Session`](crate::Session).
+pub mod info {
+    pub use crate::api::info::PeersZidBuilder;
+    pub use crate::api::info::RoutersZidBuilder;
+    pub use crate::api::info::SessionInfo;
+    pub use crate::api::info::ZidBuilder;
```

Comment:
Not in the scope of this PR but I believe `ZidBuilder` should be called `ZenohIdBuilder` since the correct type is `ZenohId`.

Reply:
Added issue to not forget https://github.com/eclipse-zenoh/zenoh/issues/996